x-pack/plugins/siem/server/lib/detection_engine/rules/update_rules_notifications.ts

```diff
@@ -45,5 +45,8 @@ export const updateRulesNotifications = async ({
     interval: ruleActions.alertThrottle,
   });
 
+  // TODO: Workaround for https://github.com/elastic/kibana/issues/53868
```

Comment:
We've created this issue for the proper fix. I'm pretty sure that issue describes the cause for SIEM.

```suggestion
  // TODO: Workaround for https://github.com/elastic/kibana/issues/67290
```